src/index.html

```diff
@@ -1,12 +1,297 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="styles/header.css">
+    <link rel="stylesheet" href="styles/product-card.css">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="/" class="logo">
+        <img
+          src="images/logo.png"
+          alt="Логотип"
+        >
+      </a>
+      <nav class="header_menu">
+        <a href="/" class="item is-active">apple</a>
+        <a href="/" class="item">samsung</a>
+        <a href="/" class="item">смартфоны и телефоны</a>
+        <a
+          href="/"
+          class="item"
+          data-qa="nav-hover"
+        >
+          ноутбуки и компьютеры
+        </a>
+        <a href="/" class="item">гаджеты</a>
```

Comment:
```suggestion
        <a href="#" class="item">гаджеты</a>
```
use `#` as default href

Reply:
+